### ISSUE TITLE
As a car pooling user, I would like to have the new  car pooling GUI to be integrated in mobility.meran.eu

### ISSUE THREAD

**Issue by rcavaliere** [idm-suedtirol/mobility.meran.eu#22]

**Comment by rcavaliere** [comment]
Please use the attached GUI elements prepared by Hell Company
[190118_GUI_car-pooling.zip](https://github.com/idm-suedtirol/mobility.meran.eu/files/2798763/190118_GUI_car-pooling.zip)

**Comment by bertolla** [comment]
@rcavaliere 
Wouldn't it be better to let Daniel R. implement this in the new version of mobility.meran.eu avoiding doing the same work twice in a short amount of time? We also have a beta version which is pointing to another realtimebus server, showing no lines right now and we risk to have conflicts in the final version.

**Comment by rcavaliere** [comment]
@bertolla no, this would take too long. The car pooling system is already implemented, it is just a matter to change some little things in the UI and to integrate the car pooling layer in the production system of mobility.meran.eu

**Comment by bertolla** [comment]
To be able to implement the new carpooling api I need the following graphical elements which are not in the current zip file:

- [ ] all blue location icons in the right format (png or jpg) in the correct size

- [ ] a lot of icons are missing, in the zip file there are 3, while in the pdf there are at least 10 new icons

**Comment by rcavaliere** [comment]
Mailed to Hell, let's wait for their feedback. In the meantime can you make some progress with this activity, so that when the icons are available we just substitute them?

**Comment by rcavaliere** [comment]
Feedback from Hell received, see attachment
[carpooling_icone.zip](https://github.com/idm-suedtirol/mobility.meran.eu/files/2890343/carpooling_icone.zip)

**Comment by bertolla** [comment]
OK, I can move on with development, but I'm still missing **the marker icons** in the correct colors like for example:
<img src="https://mobility.meran.eu/alpha/images/9_Carpooling/marker.svg" width="60"/>
<img src="https://mobility.meran.eu/alpha/images/9_Carpooling/marker_red.svg" width="60"/>
<img src="https://mobility.meran.eu/alpha/images/9_Carpooling/marker_orange.svg" width="60"/>
<img src="https://mobility.meran.eu/alpha/images/9_Carpooling/marker_green.svg" width="60"/>

**Comment by rcavaliere** [comment]
@bertolla no, we don't need these marker icons here! We just have to show users and hubs

**Comment by bertolla** [comment]
@rcavaliere 
Ok but i still need the icons in the right color and shape as shown above.

**Comment by bertolla** [comment]
There is also a little problem right now. On the screendesign there are inforamtion like for example "Attesa massima", we do not even have and on the other side information we have like, the statistics of the the carpooling service are not shown at all. I there a reason why? How should I go on?
You can see the changes I made here: https://mobility.meran.eu/alpha/realtimebus/client/?l=de#

**Comment by rcavaliere** [comment]
@bertolla it is foreseen that the API with Innovie will be integrated with additional information (e.g. number of likes), we will do this in following sprints. Thanks for the work so far, I will ask Hell for the color to be used for the icons so that you can also update the icons on the map.

**Comment by bertolla** [comment]
@rcavaliere 
Please test the version here: https://mobility.meran.eu/alpha/realtimebus/client/
I did not implement everything since we do not have all data required, and I left thins which are general to all themes the same as for others

**Comment by rcavaliere** [comment]
To do list:
- [ ] Integration in bdp-core-v2
- [ ] icon driver + passenger
- [ ] no overall statistics
- [ ] additional meta information (e.g ranking) -> enhanced Innovie API
- [ ] direct link to Innovie platform when contacting user -> sync with Innovie needed there
